### ISSUE TITLE
Discord Outdated

### ISSUE THREAD

**Issue by RushanNotOfficial** [sandipndev/melody-discord-bot#5]
Is there a new version because this version of discord.py used is quite outdated and requires an update.

**Comment by sandipndev** [comment]
I agree, I have more focus on other projects. Will look into it this weekend.

**Comment by RKNpy** [comment]
You can use pycord, its the most popular discord.py fork its maintained and have slash commands, applications menu
(updated)
